### PR TITLE
add child project council districts to summary tab

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/14305

## Testing
**URL to test:** 

because there is a change to the metadata, you will have to test this locally

**Steps to test:**

Create a project with no mapped components, but with child projects that have mapped components. Or if you are using prod data, https://localhost:3000/moped/projects/509. The summary tab has council districts

Screenshot 2023-10-24 at 9 39 07 AM

Also test a project with mapped components that also has child projects with mapped components, and a project without any child projects

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-database/metadata/tables.yaml [hunk 1]
```diff
@@ -3581,6 +3581,15 @@
...
             schema: public
+    - name: project_geography
+      using:
+        manual_configuration:
+          column_mapping:
+            project_id: project_id
+          insertion_order: null
+          remote_table:
+            name: project_geography
+            schema: public
```
Comment (johnclary) [inline moped-database/metadata/tables.yaml:3592]: thank you for setting this up!

**Review by johnclary — APPROVED** [review]
i tried to break it—i could not! it looks great and works great. i love this improvement! thank you! 🚢

File: moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryCouncilDistricts.js [hunk 2]
```diff
@@ -11,9 +11,15 @@ const getCouncilDistricts = (data) => {
 
   // flatten the array of arrays and remove empty districts
   const districtsArray = districts.flat().filter((d) => d);
+  return districtsArray;
+};
 
+const getAllCouncilDistricts = (projectGeography, childProjectGeography) => {
```
Comment (mddilley) [inline moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryCouncilDistricts.js:17]: ahh cool - i like how this code spells out what makes up the council districts (parent + child) 🚀

**Review by mddilley — APPROVED** [review]
I tested with prod data locally and with project 509, and I see all of the council districts of all of its related components on the summary page. 😎 🚢

**Pull request merged**